src/main/app/src/app/shared/confirm-dialog/confirm-dialog.component.html

```diff
@@ -3,15 +3,13 @@
   ~ SPDX-License-Identifier: EUPL-1.2+
   -->
 
-<div mat-dialog-content>
-    <mfb-form-field [field]="data.formField"></mfb-form-field>
-</div>
+<div mat-dialog-content [innerHTML]="data.melding"></div>
 <div mat-dialog-actions>
-    <button [disabled]="loading || data?.formField.formControl.invalid" mat-button color="primary" (click)="confirm()" id="confirmButton">
+    <button [disabled]="loading" mat-button color="primary" (click)="confirm()" id="confirmButton">
         <mat-icon *ngIf="loading">
             <mat-spinner diameter="20"></mat-spinner>
         </mat-icon>
-        {{ data.confirmButtonActionKey | translate }}
+        {{'actie.ja' | translate}}
     </button>
-    <button [disabled]="loading" autofocus mat-button (click)="cancel()" id="cancelButton">{{ data.cancelButtonActionKey | translate }}</button>
+    <button [disabled]="loading" autofocus mat-button (click)="cancel()" id="closeButton">{{'actie.nee' | translate}}</button>
```

Comment:
Strict genomen doet de confirmButton toch ook een close? Is een 'cancelButton' dan niet beter? Of een "yesButton" een "noButton" button?